examples/tutorial/A1_Exploration_with_Containers.ipynb

```diff
@@ -129,7 +129,8 @@
     "# in the same way `fm_modulation` returned a ``Curve`` element.\n",
     "def fm_modulation2(f_carrier=220, f_mod=110, mod_index=1, length=0.1, sampleRate=3000):\n",
     "    x = np.arange(0,length, 1.0/sampleRate)\n",
-    "    y = np.sin(2*np.pi*f_carrier*x + mod_index*np.sin(2*np.pi*f_mod*x))"
+    "    y = np.sin(2*np.pi*f_carrier*x + mod_index*np.sin(2*np.pi*f_mod*x))\n",
+    "    return hv.Curve((x, y), kdims=['Time'], vdims=['Amplitude'])"
```

Comment:
Was this line deleted accidentally somehow?  How would it ever work otherwise?

Reply:
It is in an exercise so I imagine the idea was that the user was supposed to write the return statement

Reply:
Ok, in that case presumably the added line should be deleted?